index.css

```diff
@@ -0,0 +1,90 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+nav {
+  max-width: 100%;
+  height: 40px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 20px;
+}
+
+nav .logo {
+  width: 76px;
+  height: 20px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  white-space: nowrap;
+}
+
+nav .menu-bar {
+  font-size: 30px;
+  color: #6070ff;
+}
+
+nav .drop-down {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  display: none;
+}
+
+.headline-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  max-width: 100%;
+  height: 85vh;
+  color: #344563;
+  background-color: #fff;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 1600 900'%3E%3Cdefs%3E%3ClinearGradient id='a' x1='0' x2='0' y1='1' y2='0'%3E%3Cstop offset='0' stop-color='%236070FF'/%3E%3Cstop offset='1' stop-color='%236070FF'/%3E%3C/linearGradient%3E%3ClinearGradient id='b' x1='0' x2='0' y1='0' y2='1'%3E%3Cstop offset='0' stop-color='%236070FF'/%3E%3Cstop offset='1' stop-color='%236070FF'/%3E%3C/linearGradient%3E%3C/defs%3E%3Cg fill='%23FFF' fill-opacity='0' stroke-miterlimit='10'%3E%3Cg stroke='url(%23a)' stroke-width='66'%3E%3Cpath transform='translate(-92.75 21.2) rotate(13.25 1409 581) scale(1.053)' d='M1409 581 1450.35 511 1490 581z'/%3E%3Ccircle stroke-width='22' transform='translate(-53 53) rotate(15.9 800 450) scale(1.027)' cx='500' cy='100' r='40'/%3E%3Cpath transform='translate(47.7 -159) rotate(159 401 736) scale(1.027)' d='M400.86 735.5h-83.73c0-23.12 18.74-41.87 41.87-41.87S400.86 712.38 400.86 735.5z'/%3E%3C/g%3E%3Cg stroke='url(%23b)' stroke-width='20'%3E%3Cpath transform='translate(318 -21.2) rotate(5.3 150 345) scale(0.947)' d='M149.8 345.2 118.4 389.8 149.8 434.4 181.2 389.8z'/%3E%3Crect stroke-width='44' transform='translate(-212 -132.5) rotate(190.8 1089 759)' x='1039' y='709' width='100' height='100'/%3E%3Cpath transform='translate(-318 106) rotate(31.8 1400 132)' d='M1426.8 132.4 1405.7 168.8 1363.7 168.8 1342.7 132.4 1363.7 96 1405.7 96z'/%3E%3C/g%3E%3C/g%3E%3C/svg%3E");
+  background-size: cover;
+  padding: 20px;
+  margin-block-start: 40px;
```

Comment:
- [ ] Your background image doesn't seem correctly positioned on the screen. Please try to fix it. Here's screenshots: 
your design:
![Screenshot from 2022-09-21 23-54-10](https://user-images.githubusercontent.com/91486791/191608447-82fc51e3-32c8-4a7c-8191-c7de046d584f.png)

figma:
![Screenshot from 2022-09-21 23-49-19](https://user-images.githubusercontent.com/91486791/191608040-ac536d32-9aea-4a34-b489-abbb54d01a87.png)